route/build/src/read_control.f90

```diff
@@ -247,6 +254,9 @@ SUBROUTINE read_control(ctl_fname, err, message)
    case('<KWheight>');             read(cData,*,iostat=io_error) meta_rflx(ixRFLX%KWheight         )%varFile  ! default: true (turned off if floodplain is inactive)
    case('<MCheight>');             read(cData,*,iostat=io_error) meta_rflx(ixRFLX%MCheight         )%varFile  ! default: true (turned off if floodplain is inactive)
    case('<DWheight>');             read(cData,*,iostat=io_error) meta_rflx(ixRFLX%DWheight         )%varFile  ! default: true (turned off if floodplain is inactive)
+   case('<localDOC>');             read(cData,*,iostat=io_error) meta_rflx(ixRFLX%localCC          )%varFile  ! default: faulse (turned off if tracer is inactive)
```

Comment:
Misspelling of false.

These have DOC in the name, but so far I haven't seen how DOC is hardcoded as the concentration solute. Does it really need DOC in the name here? Although maybe that's good to make it clear in the control file...

Reply:
Maybe change them to basDOC, soluteMass, soluteFlux?